README.md

```diff
@@ -100,7 +100,15 @@ XLM Wallet Address:
 - `npm install ngx-quill`
 - for projects using Angular < v5.0.0 install `npm install ngx-quill@1.6.0`
 - install `@angular/core`, `@angular/common`, `@angular/forms`, `@angular/platform-browser`, `quill`, and `rxjs` - peer dependencies of ngx-quill
-- include theme stylings: bubble.css, snow.css of quilljs in your index.html, or add them in your css/scss files with `@import` statements, or add them external stylings in your build process.
+- include theme stylings: 	**bubble.css, snow.css and core.css of quilljs** in your index.html (you can find them in `node_modules/dist`), or add them in your css/scss files  with `@import` statements, or add them external stylings in your build process.
```

Comment:
> you can find them in `node_modules/dist`

should be `node_modules/quill/dist`?

Reply:
Yep! Sorry for that, I pushed the change 👍